src/views/notifications/TradeNotification/index.tsx

```diff
@@ -65,43 +66,67 @@ export const TradeNotification = ({ isToast, data, notification }: TradeNotifica
         </Styled.OrderStatus>
       }
       slotCustomContent={
-        <Styled.Details
-          items={[
-            {
-              key: 'size',
-              label: (
-                <Styled.Label>
-                  {stringGetter({ key: STRING_KEYS.SIZE })}
-                  <OrderSideTag
-                    orderSide={SIDE === STRING_KEYS.BUY ? OrderSide.BUY : OrderSide.SELL}
-                  />
-                </Styled.Label>
-              ),
-              value: (
-                <Output type={OutputType.Asset} value={FILLED_AMOUNT} tag={marketData?.assetId} />
-              ),
-            },
-            {
-              key: 'price',
-              label: stringGetter({ key: STRING_KEYS.PRICE }),
-              value:
-                ORDER_TYPE === TradeTypes.MARKET ? (
-                  <span>{stringGetter({ key: STRING_KEYS.MARKET_ORDER_SHORT })}</span>
-                ) : (
-                  <Output
-                    type={OutputType.Fiat}
-                    value={AVERAGE_PRICE}
-                    fractionDigits={marketData?.configs?.displayTickSizeDecimals}
-                  />
-                ),
-            },
-          ]}
+        <FillDetails
+          orderSide={SIDE === STRING_KEYS.BUY ? OrderSide.BUY : OrderSide.SELL}
+          tradeType={tradeType}
+          filledAmount={FILLED_AMOUNT}
+          assetId={marketData?.assetId}
+          averagePrice={AVERAGE_PRICE}
+          tickSizeDecimals={marketData?.configs?.displayTickSizeDecimals ?? USD_DECIMALS}
         />
       }
     />
   );
 };
 
+export const FillDetails = ({
```

Comment:
nit: Move file outside of `index.tsx` into broader `TradeNotification` directory or remove `export`?